english/red-pill/red-pill.md

```diff
@@ -1,3 +1,6 @@
 An wrinkled old man with a funny hat appears and hands you a red pill and a glass of chocolate milk. 
 
-You take the pill then drink the milk and within seconds you become extremely hungry but only for tacos.
\ No newline at end of file
+You take the pill then drink the milk and within seconds you become extremely hungry but only for tacos.
+
+However, you realize that there was much more to the red pill than 
+expected. On top of craving tacos, you start seeing bizarre yellow glowing coagulating blobs floating everywhere. It might've been the [mushrooms in the chocolate milk](../mushroom/eat) or the combined effects of [the mushrooms and red pill](../the-truth).
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.